### DISCUSSION TITLE
Datenpunkt Stil basierend auf Ack flag

### DISCUSSION THREAD

**Discussion by protree** [Zefau/ioBroker.jarvis#2859] [Q&A]
Ist es möglich, den Datenpunkt Stil in jarvis basierend auf dem Ack Flag des Datenpunkts in iobroker zu ändern?

Konkretes Beispiel: Meine Heizungsthermostate haben einen Datenpunkt „settemp“. Wenn ich den schreibe ist zunächst ack=false. Bestätigt das Thermostat die Umsetzung wird der Datenpunkt mit ack=true vom Adapter geschrieben. Ich hätte das gerne als Textfarbe in jarvis abgebildet. Text der Temperatur rot für ack=false, grün für ack=true.

Vielen Dank für Eure Hilfe und vielen Dank für Jarvis!

**Comment by mcuiobroker** [comment]
evtl mit 
```
{"{val.ack} = true":{"color":"green"}}
```

**Comment by protree** [comment]
Danke für die Antwort, klappt so leider nicht. Fehler in der Browser Konsole:

`ioBroker-a51989c6"Incorrect callback function (val.ack == true) supplied to Heizung_Schlafzimmer (Temperatur)!"/.+?\(.*?\).*?/gfalsefalsefalsefalsetrue
ReferenceError: Can't find variable: val`

**Reply by mcuiobroker** [reply]
Und nur {ack}=true

**Reply by protree** [reply]
Klappt nicht, findet die Variable auch nicht. value.ack klappt auch nicht...

**Reply by mcuiobroker** [reply]
Dann als Feature eintragen

**Reply by protree** [reply]
https://github.com/Zefau/ioBroker.jarvis/issues/2860

Danke trotzdem!
